docs/apm-configuration/next-js/vercel-integration.mdx

```diff
@@ -1,64 +1,65 @@
-      <li>After 3sec, You will be redirected back to the marketplace page with Middleware integrated. </li>
-</ol>
+# Install
 
-## Essential Considerations
-<ul>
-    <li>The `instrumentation.ts` file must be located at the root of your project, not inside the `app` or `pages` directory.</li>
-    <li>In case you're using the `src` folder, please ensure that the `instrumentation.ts` file is placed within the `src` directory, alongside the `pages` and `app` directories.</li>
-    <li>If you use the `pagesExtension` <a href="https://nextjs.org/docs/pages/api-reference/next-config-js/pageExtensions" rel="nofollow noopener noreferrer" target="_blank" translate="no" color="blue"> config option </a> to add a suffix, it is essential to synchronize the instrumentation filename accordingly. For example, if you use `pagesExtension: ['ts']`, the instrumentation file should be named `instrumentation.ts`.</li>
-    <li>It's important to note that when you don't have any APIs (or Serverless functions) integrated, the Vercel platform will treat the app as a static website during deployment. Consequently, in this scenario, the `instrumentation.ts` file will not be invoked, resulting in the absence of traces or logs in your Middleware account. <br /><br />If you're looking to set up a sample Next.js project with API routes, you can use the following command in your terminal: ```npx create-next-app@latest --example api-routes```</li>
-</ul>
-<Note>You can read detailed prerequisites and essential information about OpenTelemetry in <a href="https://nextjs.org/docs/pages/building-your-application/optimizing/open-telemetry" rel="nofollow noopener noreferrer" target="_blank" translate="no" color="blue">Vercel's documentation</a>. This will be helpful for you in seamlessly integrating it into your project.</Note>
+### Step 1: Add Middleware Integration
 
-## Getting Started with Code
-Once you have successfully integrated Middleware, proceed with the following steps to incorporate it into your Vercel project and monitor real-time activity:
+1. Add the [Middleware integration](https://vercel.com/integrations/middleware/new) from the Vercel marketplace
 
-1. Install the following package that will help to send all the traces and profiling data generated from your project. This package will instrument your project based on the OpenTelemetry approach.
-<Note>Make sure you have installed the latest version of Next.js or a version greater than 13.4+, as Vercel introduced their experimental feature in that release.</Note>
-<Note>Before proceeding with the Next.js APM setup, make sure you have the **@opentelemetry/api** package installed. If it's not already installed, run the following command: `npm install @opentelemetry/api@">=1.3.0 <1.5.0"`</Note>
+2. Select your Vercel project and log into your Middleware account
 
-```
+3. Enable traces prior to deploying your project
```

Comment:
This is a necessary requirement that was somewhat hidden on the previous doc. You can technically deploy your project prior to enabling traces so I think its important to clarify that they have to enable traces at this step prior to installing the APM because users wont go back to the Vercel page until they have completed the rest of the set up.